rocks/RocksDatabase.java

```diff
@@ -260,11 +269,17 @@ public void delete() {
         private final TraversalCache traversalCache;
         private final ReasonerCache reasonerCache;
         private final SchemaGraph schemaGraph;
+        private final RocksStorage schemaStorage;
+        private long borrowerCount;
+        private boolean invalidated;
 
         private Cache(RocksDatabase database) {
```

Comment:
The `Cache` object expects the parent class `RocksDatabase` to call it in a thread safe way. Given that assumption, we no longer use any atomic variables